### PR TITLE
remove redux

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -30,13 +29,11 @@ const App = () => {
     <SafeAreaProvider>
       <RNP.Provider>
         <AuthProvider>
-          <StorageProvider>
```
Comment (wolverineks) [inline src/App.tsx:33]: lift `StorageProvider` into `index.tsx` to enable `useStorage` in this file

File: src/index.tsx [hunk 2]
```diff
@@ -70,19 +66,19 @@ const AppWithProviders = () => {
...
-            <LoadingBoundary style={StyleSheet.absoluteFill}>
-              <ThemeProvider>
-                <LanguageProvider>
-                  <CurrencyProvider>
+        <QueryClientProvider client={queryClient}>
+          <LoadingBoundary style={StyleSheet.absoluteFill}>
+            <ThemeProvider>
+              <LanguageProvider>
+                <CurrencyProvider>
+                  <StorageProvider>
```
Comment (wolverineks) [inline src/index.tsx:74]: lifted from `App.tsx`

File: src/WalletInit/WalletNameForm/WalletNameForm.tsx [hunk 3]
```diff
@@ -50,12 +49,6 @@ export const WalletNameForm = ({
   }
   const walletNameErrorText = getWalletNameError(errorMessages, validationErrors) ?? undefined
 
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  const submit = React.useCallback((ignoreConcurrentAsyncHandler as any)(() => () => onSubmit({name}), 1000)(), [
```
Comment (stackchain) [inline src/WalletInit/WalletNameForm/WalletNameForm.tsx:54]: 👀

**Review by stackchain — APPROVED** [review]
lgtm

**Pull request merged**